src/Raven.Studio/typescript/viewmodels/database/tasks/importDatabaseFromFile.ts

```diff
@@ -72,51 +72,16 @@ class importDatabaseFromFile extends viewModelBase {
             }
         });
         
-        const modelAsJson = () => {
-            const args = this.model.toDto();
-            if (!args.TransformScript) {
-                delete args.TransformScript;
-            }
-            return JSON.stringify(args);
-        };
-        
-        const fileNameProvider = () => this.importedFileName() || "Dump of Database.ravendbdump";
-        const commandEndpointUrl = (db: database) => appUrl.forServer() + appUrl.forDatabaseQuery(db) + endpoints.databases.smuggler.smugglerImport;
-        
-        this.importCommandPowerShell = ko.pureComputed(() => {
-            const db = this.activeDatabase();
-            if (!db) {
-                return "";
-            }
-            
-            const json = modelAsJson();
-            const fileName = fileNameProvider();
-
-            return `curl.exe -F 'importOptions=${json.replace(/"/g, '\\"')}' -F 'file=@.\\${fileName}' ${commandEndpointUrl(db)}`;
-        });
-        
-        this.importCommandBash = ko.pureComputed(() => {
-            const db = this.activeDatabase();
-            if (!db) {
-                return "";
-            }
-
-            const json = modelAsJson();
-            const fileName = fileNameProvider();
-            
-            return `curl -F 'importOptions=${json}' -F 'file=@${fileName}' ${commandEndpointUrl(db)}`;
-        });
-        
-        this.importCommandCmd = ko.pureComputed(() => {
-            const db = this.activeDatabase();
-            if (!db) {
-                return "";
+        this.effectiveCommand = ko.pureComputed(() => {
+            const cmdType = this.effectiveCommandType();
```

Comment:
this method can be simplified to one line: ko.pureComputed(() => this.getCommand(this.effectiveCommandType()));

Reply:
done